_posts/2021-01-06-how-to-update-to-cis-13.adoc

```diff
@@ -0,0 +1,124 @@
+  <<compatibility_table,version compatibility table>> you can use as a reference to know which Gruntwork Repo version
+  tag is compatible with CIS AWS 1.3.0, as well as the manuals step you need to perform to achieve said compliance.
+
+
+[[core_concepts]]
+== Core Concepts
+
+=== Background
+
+The 1.3.0 version of the CIS AWS Foundations Benchmark was release in September of 2020. You can refer to the
+https://www.cisecurity.org/benchmark/amazon_web_services/[CIS website] where you can download the latest version of the
+Benchmark (as well as all the previous version). The latest version introduces several new recommendations, and this guide
+will walk you through implementing these with help from the Gruntwork's Infrastructure as Code Library,
+so that your code becomes fully compliant with the 1.3.0 version of the framework.
+
+
+[[deployment_walkthrough]]
+== Deployment walkthrough
+
+=== Step 1: Update references to the Gruntwork Infrastructure as Code Library
+
+In order to update to the CIS AWS Foundations Benchmark v1.3.0, you need to update your references to the Gruntwork
+Infrastructure as Code Library to use a compatible version. We (Gruntwork) have reviewed and updated all the modules in the
+library for compatibility with the new version of the Benchmark. As a customer, you need to update to
+the proper versions of the Gruntwork library to pick up the fixes/changes that were made to be compatible. Refer to
+https://gruntwork.io/guides/foundations/how-to-use-gruntwork-infrastructure-as-code-library/#updating[the
+"Updating" section of "How to use the Gruntwork Infrastructure as Code Library"] for instructions on how to update the
+versions in your code.
+
+For the vast majority of the repos, the only change that will be necessary is a version number bump, but several repos
+require more extensive code changes and state migrations. To upgrade without downtime and data loss, **you MUST follow
+the migration instructions in the release notes in each repo to know what changes need to be made to update to the new
+version.**
+
+[.exceptional]
+IMPORTANT: Gruntwork follows
+https://gruntwork.io/guides/foundations/how-to-use-gruntwork-infrastructure-as-code-library/#versioning[semantic
+versioning]. For any pre-1.0 modules, this means that version updates to the minor version are considered backwards
+incompatible releases for any version updates prior to 1.0.0 release. Make sure to read the release notes for the
+relevant modules any time you are updating minor versions! Note that you will want to read the release notes for each
+minor version that is updated (e.g., if you are going from v0.5.x to v0.9.x, you will want to read the notes for v0.6.0,
+v0.7.0, v0.8.0, and v0.9.0 to get the full list of backwards incompatible updates).
+
+The following table provides a summary of all the relevant Gruntwork AWS modules and the respective versions that are
+compatible with CIS AWS 1.3.0:
+
+[[compatibility_table]]
```

Comment:
Reminder to update these versions when we complete the last few items for 1.3 (e.g., the NACLs stuff).